src/components/cards/TradeCard/GasReimbursement.vue

```diff
@@ -51,7 +51,7 @@ export default defineComponent({
     const { t } = useI18n();
     const BAL = '0xba100000625a3754423978a60c9317c58a424e3d';
 
-    const eligibleAssetMeta = eligibleAssetList[appNetwork.networkName];
+    const eligibleAssetMeta = eligibleAssetList[appNetwork.networkName] ?? {};
     const eligibleAssets = Object.fromEntries(
```

Comment:
minor - what if we do a condition on the `eligibleAssets` instead? (just to save us from doing the computation which we know won't happen)

```
const eligibleAssets = eligibleAssetMeta == null ? {} : Object.fromEntries(...)`
```

Reply:
I think this effect would be so small to be unnoticeable tbh. In any case it increases computation on the happy path which will be much more common.